source/beamline/AMPVControl.h

```diff
@@ -132,12 +132,16 @@ class AMReadOnlyPVControl : public AMControl {
 
 protected slots:
 	/// Handles the low limit PV changing. Converts the limit before forwarding the signal on.
-	virtual void onLowLimitPVValueChanged(double newLowLimit);
+	virtual void setLowLimitValue(double newLowLimit);
```

Comment:
Pure semantic question, but why change from onLimitChanged to setLimit?  From what I can tell this is being primarily called via signals which would use the onXChanged style.